Core/Graphics/Source/FrameBuffer.cpp

```diff
@@ -106,6 +106,9 @@ namespace Babylon
 
         // This dummy draw call is here to make sure that the view is cleared
         // if no other draw calls are submitted to the view.
-        encoder->touch(m_viewId.value());
+        if (doTouch)
+        {
+            encoder->touch(m_viewId.value());
+        }
```

Comment:
This is an odd way to do this. Maybe just move this code into another function and call it only when touch is warranted? Or maybe just call the code without another function since it's so small anyways.

Reply:
I agree it's a bit odd. After some more thoughts, I still think it's the best way here. I'm against copy/paste to a new function or inlined code because of code divergence that can lead to difficult times when debugging. And the boolean parameter is determined at compile time. I believe moving that boolean to a parameter will give the same binary code with Release/O3 but I have no guarantee.

Reply:
I don't mean copy and paste to a new function. I mean split the function in half. There should be no code divergence. Anyways, it's not a big deal.